src/OrbitGl/MockBatcher.h

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_MOCK_BATCHER_H_
+#define ORBIT_GL_MOCK_BATCHER_H_
+
+#include <limits>
+
+#include "BatcherInterface.h"
+
+namespace orbit_gl {
+
+static bool IsInBetween(float point, float min, float max) { return point >= min && point <= max; }
+
+static bool IsInsideRectangle(Vec2 point, Vec2 start, Vec2 end) {
+  for (int i = 0; i < 2; i++) {
+    if (!IsInBetween(point[i], start[i], end[i])) {
+      return false;
+    }
+  }
+  return true;
+}
```

Comment:
Nice. Nit: nodiscard

Reply:
And also `inline` instead of `static` because it's in a header. BTW, putting the implementation into a cpp file would be even better. But your call.